src/PostList.php

```diff
@@ -8,13 +8,18 @@
  * @todo Do all lists of any object have the same format? If so, this could
  *   become just 'List'.
  */
-class PostList implements \ArrayAccess, \Countable
+class PostList implements \IteratorAggregate, \ArrayAccess, \Countable
 {
     /**
      * @var \Lullabot\Parsely\Post[]
      */
     private $data;
 
+    public function getData(): array
+    {
+        return $this->data;
```

Comment:
Having a method here gives us the option to implement paging in the future without a huge b/c break (or hopefully none at all).